### DISCUSSION TITLE
Do I need to use Vercel to use preview mode?

### DISCUSSION THREAD

**Discussion by tylermcrobert** [vercel/next.js#13650] [Help] [answered]
I've got an older nextjs site that's currently on Netlify. If I would like to update this site to use preview mode, do I need to switch to Vercel? I seemingly can't call `res.setPreviewData({})` on anything other than Vercel.

**Comment by Timer** [comment, marked as answer]
You can use Preview Mode on any host that supports `next start`, or a serverless-compatible host like Vercel or AWS (via [Serverless Framework](https://github.com/danielcondemarin/serverless-next.js)).

Besides Vercel (the platform Next.js was made for), you could host apps with Preview Mode on Heroku, Google Cloud: Cloud Run, AWS Elastic Beanstalk, Render, etc.

Netlify is a static host, so you'll need to switch providers to leverage Preview Mode.
